pom.xml

```diff
@@ -187,6 +188,40 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+				<groupId>kr.motd.maven</groupId>
+				<artifactId>os-maven-plugin</artifactId>
+				<version>1.6.2</version>
```

Comment:
Can you make it so the plugin version is defined as a property. This makes life easier when we start to  automate the CI/CD aspects.